packages/meditrak-app/app/sync/syncMigrations.jsx

```diff
@@ -46,6 +46,13 @@ const migrations = {
   9: async (synchroniser, setProgressMessage) => {
     await resyncRecordTypes(synchroniser, setProgressMessage, ['survey_screen_component']);
   },
+  // Resync all users so that the new user entity permission structure comes through
+  10: async (synchroniser, setProgressMessage) => {
+    await resyncRecordTypes(synchroniser, setProgressMessage, ['user_account']);
+  },
+  11: async (synchroniser, setProgressMessage) => {
+    await resyncRecordTypes(synchroniser, setProgressMessage, ['user_entity_permission']);
+  },
```

Comment:
Oooohhh I didn't know we could do this! This is awesome, it means we don't have to force users to do a full resync when we add those models, just syncing down the new data :smile: Great stuff!